src/Playground.res

```diff
@@ -1214,17 +1214,10 @@ module ControlPanel = {
           ("code", editorCode.current->LzString.compressToEncodedURIComponent),
         )->ignore
 
-        let querystring = Belt.Array.reduce(params, "", (acc, next) => {
-          let (key, value) = next
+        let querystring =
+          params->Js.Array2.map(((key, value)) => key ++ "=" ++ value)->Js.Array2.joinWith("&")
```

Comment:
Why `Js.Array2.map` instead of `Belt.Array.map`?
Belt is much [faster](https://johnridesa.bike/benchmark-bs/#list-array-map), obviously.

And... I see no chance of JS speeding up. (by looking at its polyfill)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#polyfill

Reply:
good question

Reply:
> https://rescript-lang.org/docs/manual/latest/api/js

This might be the answer, but I am also tempted to vote for Belt.Array.map for the sake of the performance.

Reply:
Either way's fine! The performance and readability gains of removing `reduce` here is enough that this doesn't matter. But yes, like wontheone1 linked, whenever possible we try to stay idiomatic and not carry a runtime. Here it's applied a bit inconsistently because some helpers don't exist in Js (e.g. `keepMap`).

If you're doing interop (like this repo is gonna do more of), Use the `Js.*` namespace. Otherwise `Belt` is indeed pretty good. The `map` is really just a for loop inside.